auction-app-frontend/src/constants.js

```diff
@@ -35,12 +35,17 @@ export const ROUTE_PATHS = {
   ABOUT_US: "/about-us",
   PRIVACY_POLICY: "/privacy-policy",
   TERMS_AND_CONDITIONS: "/terms-and-conditions",
+  PRODUCT_DETAILS: "/shop/:id",
```

Comment:
If route has dynamic parameter, avoid setting it in constants 
`${ROUTES.SHOP}/:id`, write it like this, it's better